packages/com_importer/site/importer.php

```diff
@@ -9,9 +9,21 @@
 // No direct access
 defined('_JEXEC') or die;
 
-$user = JFactory::getUser();
+$user			= JFactory::getUser();
+$userAuthorised	= $user->authorise('core.admin', '');
```

Comment:
Are these permissions specific to import tool or are you using Joomla's permissions ? Need to be imported specific ACL.

Reply:
These are not specific to Importer Tool. We haven't yet implemented ACL in our Tool. For now, I have used Joomla's permission.

Reply:
Ok. For now, remove this since it will be a problem when integrating with other extensions. It wont always be a super user using it. 

I've added an issue for ACL management, lets discuss and implement that later.